routes/demographics.py

```diff
@@ -85,15 +142,33 @@ def get_data_for_community(community):
         for field in fields:
             reformatted_results[c][field] = results[c][field]
 
+    # apply population threshold
+    if (
+        reformatted_results[community]["total_population"]
+        - reformatted_results[community]["pct_under_18"]
+    ) < 50:
+        return render_template("/403/pop_under_50.html"), 403
```

Comment:
```suggestion
    totalPopulation = reformatted_results[community]["total_population"]
    percentUnder18 = reformatted_results[community]["pct_under_18"]
    populationUnder18 = totalPopulation * (percentUnder18 / 100)
    adultPopulation = round(totalPopulation - populationUnder18)

    # apply population threshold
    if adultPopulation < 50:
        return render_template("/403/pop_under_50.html"), 403
```

Looks like you did the same thing I did at first when doing this on the NCR side (subtracting the raw percentage instead of the population represented by the percentage), which is why this jumped out at me. Easy fix though!

Reply:
Ah yes! Good catch! and +1 for lowerCamelCase 🐫  😃

Reply:
Dang, actually I shouldn't have been using lowerCamelCase here! My head was full of JavaScript. lowerCamelCase is considered not-very-Python. Feel free to change these to snake_case!!

Reply:
Doesn't bother me really, the code works either way! What do you think is best?

Reply:
We should probably change these variables to snake_case so as not to incur the wrath of others 😅 Sorry for the mess!

Reply:
done! 🐍